install.sh

```diff
@@ -196,12 +196,20 @@ if [ "$1" = "install" ] || [ "$1" = "config" ]; then
   if [ "$_distro" = "Debian" ]; then #Help Debian cert problem.
     sed -i -e 's#CONFIG_SYSTEM_TRUSTED_KEYS="debian/certs/test-signing-certs.pem"#CONFIG_SYSTEM_TRUSTED_KEYS=""#g' .config
   fi
-  yes '' | make oldconfig
+  yes '' | make ${llvm_opt} oldconfig
   msg2 "Done"
 
   # apply linux-tkg patching script
   _tkg_srcprep
 
+  # source cpuschedset since _cpusched isn't set
+  source "$srcdir"/cpuschedset
+
+  # Uppercase characters are not allowed in source package name for debian based distros
+  if [ "$_cpusched" = "MuQSS" ] && [ "$_distro" = "Debian" ] || [ "$_distro" = "Ubuntu" ]; then
```

Comment:
I believe this evaluates as:
```bash
(_cpusched == "MuQSS" && _distro == "MuQSS") || _distro == "Ubuntu"
```
If so, this code would have the unintended consequence of always setting `_cpusched="muqss"` on Ubuntu. Since this codebase does seem to the allow the usage of `[[`, I might change this to:
```bash
  if [[ "$_cpusched" = "MuQSS" && ( "$_distro" = "Debian" || "$_distro" = "Ubuntu" ) ]]; then

Reply:
Oh, it looks like @Tk-Glitch beat me to this, haha.